dependencies.gradle

```diff
@@ -73,13 +71,13 @@ ext {
     appCompatVersion = "1.3.1"
     constraintLayoutVersion = "2.0.4"
     coreKtxVersion = "1.5.0"
-    windowVersion = "1.0.0-alpha01"
+    windowAlphaVersion = "1.0.0-alpha01"
```

Comment:
For future us, It should be informative to add a comment there about why that new dependency is there.